names.md

```diff
@@ -1,12 +1,12 @@
 subl# This is a list of all the names of people working on this project
 
 # Format -   Full name - nickname (optional)
-Rucha Rathor
+Rucha Rathorgit sata
```

Comment:
You created your branch based off the color branch.  You should create the new branch based off the master branch